src/lib/message.ts

```diff
@@ -0,0 +1,23 @@
+import { config } from "./config";
+import { Context } from "semantic-release";
+
+export async function preparemessage(config: config, context: Context) {
+  const { env, nextRelease, logger } = context;
+  logger.log('prepare message.');
+  if (!env.DISCORD_CUSTOM_MESSAGE || !config.custom_message) {
```

Comment:
Inverted condition : you will enter in this if only if there is no custom message.